testsuite/forge_wrapper_core/process.py

```diff
@@ -55,3 +56,6 @@ def atexit(self, callback: Callable[[], None]) -> None:
 
     def user(self) -> str:
         return get_current_user()
```

Comment:
nit just replace this with the contents of the beelow function, dont need two functinos here